### ISSUE TITLE
Avatar Voices

### ISSUE THREAD

**Issue by rohrlach** [upstage-org/limelight#189]
When a player is holding an avatar and that avatar is on the stage - Whatever the player says in chat should be synthesized by HTML5 text-to-speech

**Comment by rohrlach** [comment]
Scope of this issue: When player is holding avatar, their speech is synthesized.

**Comment by rohrlach** [comment]
NOTE: There is only one voice, there is no voice assignment.  The browser synthesizes however it wants to - This should pass if it "Actually works" more options to come later

**Comment by helenvarley** [comment]
in chrome: i heard both a "male" (deeper) and a "female" (higher) voice at the same time. the male voice said "hello" once, the female voice said it 3 times, and the text appeared twice in the chat. (in my other browser window it only appeared once).
in firefox: male voice said hello once, it appeared once in the chat

**Comment by rohrlach** [comment]
Noted

**Comment by rohrlach** [comment]
Haven't been able to recreate the double voice issue as yet.

**Comment by helenvarley** [comment]
neither can i. but in chrome, i got a female voice, & in firefox, a male voice. it should be the same voice in all browsers.

**Comment by rohrlach** [comment]
Oh I see. No, not necessarily - As mentioned above, the browser will use whatever configuration it has (Which may differ between browsers) 

Again, all I'm looking for here is that the dialog is synthesized somehow

**Comment by helenvarley** [comment]
ok; but is the ultimate aim to have multiple voices that we can select from, & have the same voice heard whatever browser you are using? because there are artistic reasons for chosing particular voices, so if different browsers deliver different voices then that is not really ideal ...

**Comment by vickis** [comment]
- if the browser is speaking and the voice sounds the same whatever the avatar there will be no sense of conversations between two or more avatars (and of course the voice being part of the 'costume' is important :)

**Comment by rohrlach** [comment]
Yes, obviously this is the end goal. This ticket focuses on "can we make it speak" with a subsequent ticket for "can we make voices distinct"

**Comment by vickis** [comment]
Perfect :)
